.npmrc

```diff
@@ -0,0 +1 @@
+shell-emulator=true
```

Comment:
Guess no need to add a new line here as Prettier does not do that automatically. 🤔